.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Master CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master    
+  workflow_dispatch:    
+
+jobs:
+  test-get-kernel-sources-action:
+    runs-on: actuated-4cpu-12gb
```

Comment:
I use an x86 small node for this test.

Reply:
I think Arm would make sense to match how it's used downstream?

Reply:
Yes, but it should work the same on x86; anyway i can switch to arm. Can you drop me the link to the list of available `runs-on` for actuated? I struggle to find it :)

Reply:
It's just a convention, there is no set list.

In this case, change actuated- to acuated-arm64-

https://docs.actuated.dev

Reply:
Done!